modules/heat_conduction/tutorials/introduction/tests

```diff
@@ -42,6 +42,7 @@
       command = 'therm_soln_compare.py'
       imagediff = 'therm_soln_compare.png'
       required_python_packages = 'pandas matplotlib'
+      valgrind = none
```

Comment:
what s the concern here? mpl not vlgrind clean?

Reply:
running python doesn't work with valgrind